src/components/App/NotificationSnackbar.js

```diff
@@ -51,7 +50,7 @@ class NotificationSnackbar extends PureComponent {
     snackbarAction = (key) => {
         const { classes } = this.props;
         return (
-            <Fragment>
+            <>
```

Comment:
We only need to use a `<>` when there is more than 1 element being returned. In this case, since it's just `<IconButton>`, we can remove the `<>`

This is because the return statement can only return one element. So if there multiple elements next to each other, we wrap them in `<>`.
| 1 Element | Multiple Elements |
| --- | --- |
| Doesn't need `<>` | Needs `<>` |
| <img width="453" alt="image" src="https://user-images.githubusercontent.com/29494270/158474051-89f50c4b-2609-469e-9308-01e8db316295.png"> | <img width="606" alt="image" src="https://user-images.githubusercontent.com/29494270/158474113-32afad97-a8aa-4643-859e-1775934d0998.png"> |